util/rate_limiter.cc

```diff
@@ -111,7 +111,8 @@ void GenericRateLimiter::Request(int64_t bytes, const Env::IOPriority pri,
     std::chrono::microseconds now(NowMicrosMonotonic(env_));
     if (now - tuned_time_ >=
         kRefillsPerTune * std::chrono::microseconds(refill_period_us_)) {
-      Tune();
+      Status s = Tune();
+      assert(s.ok());  //**TODO: What to do on error?
```

Comment:
`PermitUncheckedError()` is the call to make for `Status`es that we haven't decided how to handle yet.